diff -r 4f72a0eabb16 Makefile.user.default
--- a/Makefile.user.default	Sun Dec 18 15:08:37 2016 +0200
+++ b/Makefile.user.default	Fri Dec 30 02:09:10 2016 +0100
@@ -93,13 +93,13 @@
 # GCC-related stuff
 # for yagarto or the official ARM toolchain use ARM_ABI=none-eabi otherwise use ARM_ABI=elf
 ARM_ABI=none-eabi
-GCC_VERSION=-4.8.3
+GCC_VERSION=-5.4.1
 DIETLIB_VERSION=0.33
 
 NEWLIB_OPT_LEVEL=O3
 DIETLIBC_OPT_LEVEL=Os
 
-ARM_PATH=~/gcc-arm-none-eabi-4_8-2013q4
+ARM_PATH=/usr
 ARM_BINPATH=$(ARM_PATH)/bin
 ARM_LIBGCC_PATH=$(ARM_PATH)/lib/gcc/arm-$(ARM_ABI)/$(subst -,,$(GCC_VERSION))
 NEWLIB_PATH=$(SRC_DIR)/libs/arm-$(ARM_ABI)-$(NEWLIB_OPT_LEVEL)
